### ISSUE TITLE
unused_variable: false positive for delete report

### ISSUE THREAD

**Issue by mbtools** [abaplint/abaplint#2226]
```abap
REPORT zfoobar.
DATA gv_prog TYPE string.
DELETE REPORT gv_prog.
```

/zfoobar.prog.abap[2, 6]: Variable "gv_prog" not used(unused_variables)

**Comment by larshp** [comment]
thanks 👍

**Comment by mbtools** [comment]
One more

```abap
REPORT zfoobar.
DATA gv_prog TYPE string.
DATA gt_tpool TYPE textpool_table.
INSERT TEXTPOOL gv_name FROM gt_tpool.
```

/zfoobar.prog.abap[2, 6]: Variable "gv_prog" not used(unused_variables)
/zfoobar.prog.abap[3, 6]: Variable "gt_tpool" not used(unused_variables)

**Comment by larshp** [comment]
@mbtools try again

**Comment by larshp** [comment]
closing, guess it works
